bin/firefox-driver.js

```diff
@@ -43,4 +51,21 @@ if (shouldStart) {
   setInterval(() => {}, 100);
 }
 
+function getResults(driver) {
+  driver
+    .findElement(By.id("mocha-stats"))
+    .getText().then(results => {
+      console.log("results ", results);
+      const match = results.match(/failures: (\d*)/);
+      const resultCode = parseInt(match[1], 10) > 0 ? 1 : 0;
+      process.exit(resultCode);
+    });
+}
+
+if (isTests) {
+  const driver = start();
+  driver.get("http://localhost:8003");
+  setTimeout(() => getResults(driver), 5000);
```

Comment:
Once we add more tests it'll surely take longer than 5 seconds to run. But this is a good enough thing for now and this is easy to improve later :) I could have sworn selenium had a way to "wait" until an element appears on the page or something, but it's something we can talk about later.